prowler/providers/aws/services/kms/kms_service.py

```diff
@@ -73,7 +73,7 @@ def __get_key_rotation_status__(self):
         logger.info("KMS - Get Key Rotation Status...")
         for key in self.keys:
             try:
-                if "EXTERNAL" not in key.origin:
+                if "EXTERNAL" not in key.origin and "AWS" not in key.manager:
```

Comment:
```suggestion
                if "EXTERNAL" not in key.origin and key.manager == "CUSTOMER":
```

I prefer to set `CUSTOMER`. What do you think?

Reply:
The output would be the same, so I prefer leaving it as it is.